std/haxe/io/BigBuffer.hx

```diff
@@ -0,0 +1,266 @@
+package haxe.io;
+
+import haxe.errors.NotImplemented;
+
+/**
+	TODO:
+	This is an attempt to design a cross-platform API for big byte buffers (more than 2GB)
+	without any unnecessary allocations.
+**/
+class BigBuffer {
```

Comment:
I guess this is not used in the APIs? Also endianness should be configurable.

Reply:
Yes, an intermediary `Bytes` instance has to be used to pass bytes between APIs and `BigBuffer`.
Added `endian` property.